core/src/err/mod.rs

```diff
@@ -1000,6 +1000,12 @@ pub enum Error {
 
 	#[error("This record access method does not allow signin")]
 	AccessRecordNoSignin,
+
+	#[error("Cannot construct a Record ID with table: {0}")]
+	InvalidThingTb(String),
+
+	#[error("Cannot construct a Record ID with id: {0}")]
+	InvalidThingId(String),
```

Comment:
Should we use the `Error::IdInvalid` variant, and just put the whole Record ID in there? I'm not sure how much this benefits, but it does create more error variants...

Reply:
IdInvalid does not describe an invalid table name however. We could use that and then introduce a new `TbInvalid` variant though?

Reply:
Ah yeah good point. Maybe that does work then yeah.